src/pages/blog-post.css

```diff
@@ -0,0 +1,98 @@
+@import '../styles/base.css';
+
+h3,
+h4,
+h5,
+h6 {
+  margin-top: 1rem;
+}
+
+/*
+ * Alternative `color`s:
+ * - #673AB7
+ * - #A23649
+ * - #C62D47
+ */
+p code,
+li code,
+p a code,
+li a code {
+  background: #e4e4e4;
+  border-radius: 3px;
+  color: var(--hot-pink);
+  font-size: 1rem;
+  font-weight: 400;
+  margin: 0 2px;
+  padding: 2px 1px;
+}
+
+pre {
+  font-size: 1rem;
+  margin-bottom: 1.45rem !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/app/jessestuart/jesses.io/pullRequest?prid=4576429)